### ISSUE TITLE
Create new release with Go > 1.20.11

### ISSUE THREAD

**Issue by muffl0n** [akamai/terraform-provider-akamai#522]
Grype is finding these CVEs cause of an outdated Go version used:
```
NAME    INSTALLED  FIXED-IN  TYPE       VULNERABILITY   SEVERITY 
stdlib  go1.20.11            go-module  CVE-2023-45285  High      
stdlib  go1.20.11            go-module  CVE-2023-39326  Medium
```

Could you please release a new provider version with an updated Go version?

Thank you!

* https://nvd.nist.gov/vuln/detail/CVE-2023-45285
* https://nvd.nist.gov/vuln/detail/CVE-2023-39326

**Comment by psnoch-akamai** [comment]
Hi @muffl0n ,

Thank you for raising this ticket. We plan to release v5.6.0 which includes migration to go 1.21.

Best regards,
Pawel

**Comment by lkowalsk-akamai-com** [comment]
Hi, this was addressed in recent 5.6.0 release.
